openreview/conference/templates/programchairWebfield.js

```diff
@@ -959,31 +959,54 @@ var displayStatsAndConfiguration = function(conferenceStats) {
 
   // Timeline
   var invitationMap = conferenceStatusData.invitationMap;
-
   html += '<div class="col-md-8 col-xs-12">'
   html += '<h4>Timeline:</h4><ul style="padding-left: 15px">';
-  html += renderInvitation(invitationMap, SUBMISSION_ID, 'Paper Submissions')
-  html += renderInvitation(invitationMap, REVIEWERS_ID + '/-/' + BID_NAME, 'Reviewers Bidding')
+  var notDatedElements = [];
+  var datedElements = [];
+  var pushToDatedArrays = function(invitationMap, id, name, dated, notDated){
```

Comment:
You don't need to pass references dated and notDated to this function, you can just push to the arrays defined in the outer scope.